src/main/java/com/aws/greengrass/mqtt/bridge/MQTTBridge.java

```diff
@@ -125,10 +130,10 @@ public void startup() {
         }
 
         try {
-            mqttClient = new MQTTClient(bridgeConfig.getBrokerUri(), bridgeConfig.getClientId(),
-                    mqttClientKeyStore, executorService);
-            mqttClient.start();
-            messageBridge.addOrReplaceMessageClientAndUpdateSubscriptions(TopicMapping.TopicType.LocalMqtt, mqttClient);
+            localMqttClient = localMqttClientFactory.createLocalMqttClient();
```

Comment:
this will not currently enable switching versions right? This code would only be called once right now I believe.

If so, that's fine, just want to make sure I'm on the same page.

Reply:
when version conf is changed, bridge will reinstall and this would get called again

Reply:
https://github.com/aws-greengrass/aws-greengrass-mqtt-bridge/blob/1540f3c7c0dfc6801da97a5eaa8259bde216a31e/src/main/java/com/aws/greengrass/mqtt/bridge/BridgeConfig.java#L119

Reply:
OK, that's not ideal because restart/reinstall will stop the bridge which will have it unsubscribe from everything including cloud topics which we have no reason to unsubscribe from just to change our mqtt version.

I'm ok with this for now, but consider how we may do it dynamically.

Reply:
yeah that's fair, will do as a follow-up 👍